### PR TITLE
Adds ingress.path documentation

### PR DESCRIPTION
**Pull request by bantonj**
#### What this PR does / why we need it:
When using gce-ingress in Google Cloud the ingress path needs to be set to /*. This note may help others in the future.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed

### CODE REVIEW

File: stable/atlantis/README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
...
 | `storageClassName`                          | Storage class of the volume mounted for the Atlantis data directory.                                                                                                                                                                                                                                      | n/a     |
 | `tlsSecretName`                             | Name of a Secret for Atlantis' HTTPS certificate containing the following data items `tls.crt` with the public certificate and `tls.key` with the private key.                                                                                                                                            | n/a     |
+| `ingress.path`                             | Path to use in the `Ingress`. Should be set to `/*` if using gce-ingress in Google Cloud.                                                                                                                                                                                                              | `/`     |
```
Comment (lkysow) [inline stable/atlantis/README.md]: if we're going to document `ingress` we should document all the parameters:
* ingress.enabled
* ingress.annotations
* ingress.path
* ingress.host
* ingress.tls

**Review by lkysow — CHANGES_REQUESTED** [review]

File: stable/atlantis/README.md (hunk 1 above)
Comment (bantonj) [reply stable/atlantis/README.md]: I've added the additional ingress parameters.

**Review by lkysow — COMMENTED** [review]
You need to bump the version in Chart.yaml so the linting passes.

**Review by lkysow — APPROVED** [review]

**Pull request merged**